main.go

```diff
@@ -23,225 +23,92 @@
 package main
 
 import (
-	"bytes"
-	"errors"
 	"fmt"
-	"net/url"
+	"io/ioutil"
+	"log"
 	"os"
-	"os/exec"
-	"regexp"
 	"strings"
 
-	log "github.com/Sirupsen/logrus"
-	"github.com/codegangsta/cli"
+	"github.com/uber/go-torch/pprof"
+	"github.com/uber/go-torch/renderer"
 
-	"github.com/uber/go-torch/graph"
-	"github.com/uber/go-torch/visualization"
+	gflags "github.com/jessevdk/go-flags"
 )
 
-type torcher struct {
-	commander
+// options are the parameters for go-torch.
+type options struct {
+	PProfOptions pprof.Options
+	File         string `short:"f" long:"file" default:"torch.svg" description:"Output file name (must be .svg)"`
+	Print        bool   `short:"p" long:"print" description:"Print the generated svg to stdout instead of writing to file"`
+	Raw          bool   `short:"r" long:"raw" description:"Print the raw call graph output to stdout instead of creating a flame graph; use with Brendan Gregg's flame graph perl script (see https://github.com/brendangregg/FlameGraph)"`
```

Comment:
This isn't printing in the correct format anymore. The format which can be fed directly into the flamegraph script is like

```
function1;function2;function3 3
function4;function5 6
```

I know that there is a least one application at Uber which uses the raw output from go-torch and pipes it directly into the flamegraph script.